### PR TITLE
Add support for python 3.10

### PR DESCRIPTION
**Pull request by patrick91**
This PR add tests for python 3.10 and fixes a small issue when running Strawberry on python 3.10

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: x64
 
-      - run: pip install poetry
+      - run: pip install poetry==1.2.0a1
```
Comment (BryceBeagle) [inline .github/workflows/test.yml:26]: Maybe use a build matrix for Poetry version?

Comment (patrick91) [reply .github/workflows/test.yml:26]: yup, that's an option, but I hope we can just update poetry before merging this PR, there's more dependencies that aren't ready for 3.10 :)

File: pyproject.toml [hunk 2]
```diff
@@ -13,8 +13,17 @@ repository = "https://github.com/strawberry-graphql/strawberry"
...
     "Topic :: Software Development :: Libraries",
-    "Topic :: Software Development :: Libraries :: Python Modules"
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```
Comment (patrick91) [inline pyproject.toml:26]: @jkimbo do these change make sense to you?

Comment (patrick91) [reply pyproject.toml:26]: poetry adds them automatically: https://python-poetry.org/docs/pyproject/#classifiers

so maybe we don't the ones for python versions

Comment (patrick91) [reply pyproject.toml:26]: image
yes, I'll remove the ones for python version :)

File: pyproject.toml [hunk 3]
```diff
@@ -81,6 +90,7 @@ types-freezegun = "^1.1.0"
 types-chardet = "^0.1.5"
 types-certifi = "^0.1.4"
 types-aiofiles = "^0.1.9"
+Jinja2 = "^3.0.1"
```
Comment (BryceBeagle) [inline pyproject.toml:93]: What is this new dependency for?

Comment (patrick91) [reply pyproject.toml:93]: no idea, I think there’s something broken with poetry :D will revert when it works again :)

File: pyproject.toml [hunk 4]
```diff
@@ -41,6 +44,7 @@ pydantic = {version = "<2", optional = true}
 python-multipart = "^0.0.5"
 sanic = {version = ">=20.12.2,<22.0.0", optional = true}
 aiohttp = {version = "^3.7.4.post0", optional = true}
+uvloop = "^0.16.0"
```
Comment (BryceBeagle) [inline pyproject.toml:47]: I see that the commit says "temp", but what's this for?

Comment (BryceBeagle) [reply pyproject.toml:47]: Oh wait, this isn't uvicorn

Comment (patrick91) [reply pyproject.toml:47]: yeah, uvloop < 0.16 doesn't work on python 3.10, not sure why it doesn't automatically get installed

File: .devcontainer/post-install.sh [hunk 5]
```diff
@@ -1,3 +1,2 @@
 poetry install
 pre-commit install
-pre-commit
```
Comment (patrick91) [inline .devcontainer/post-install.sh:3]: this was not needed :)

tbf we should update the dev container, but can do in another PR

File: .github/workflows/test.yml [hunk 6]
```diff
@@ -54,6 +54,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: pip install poetry
+      - run: poetry config experimental.new-installer false
```
Comment (patrick91) [inline .github/workflows/test.yml:57]: this is because poetry new installer is broken with python 3.10 :(

File: pyproject.toml [hunk 7]
```diff
@@ -13,8 +13,11 @@ repository = "https://github.com/strawberry-graphql/strawberry"
 documentation = "https://strawberry.rocks/"
 
 classifiers = [
+    "Development Status :: 5 - Production/Stable",
```
Comment (patrick91) [inline pyproject.toml:16]: @jkimbo @BryceBeagle happy to add this? :D

Comment (jkimbo) [reply pyproject.toml:16]: Yep I think so!

**Review by jkimbo — APPROVED** [review]

**Comment by codebyaryan** [comment]
This is super cool! @patrick91

**Review by BryceBeagle — APPROVED** [review]
